docs/meetings/team/2022_10_25.md

```diff
@@ -0,0 +1,86 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 9:30 am
+
+**End Time** 10:50 am
+
+**Location** Discord
+
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Veronica
+2. Edouard
+3. Logan
+4. Matt
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- Dad joke of the day
+
+## Announcements
+
+- IP Agreement ASAP Due Tomorrow
+
+- Gannt Chart (this week)
+
+- GitStats, team needs to get full marks
+
+- Assets need to be requested from Arist
+
+- Logan will not be availble at the next meeting and will unavailable for the next week
+  
+## Last Week (Restrospective)
+
+- Meeting went okay
+
+- Thanks for submitting the report @Matt and @Veronica!
+  - Did not realized we needed to answer the class questions. Thanks @Matt @Veronica!
+
+- Last week work go pushed to the end of the week
+  - Group was okay with what happened
+  - Having an early deadline helped
+
+- We should be better at planning
```

Comment:
Planning ahead will be a bit now that requirements gathering is complete and development begins